tasks/master.yml

```diff
@@ -2,7 +2,7 @@
   lineinfile:
     state: present
     backrefs: yes
-    dest: "{{ pgsqlrep_data_path }}/postgresql.conf"
+    dest: "{{ pgsqlrep_config_path }}/postgresql.conf"
```

Comment:
I don't see where this variable is defined. The path to this file on RHEL7 is `/var/lib/pgsql/9.6/data/postgresql.conf`.